components/server/src/server.rs

```diff
@@ -823,8 +823,12 @@ impl<ER: RaftEngine> TiKVServer<ER> {
 
         // Start resource metering.
         let resource_metering_cpu_recorder = resource_metering::cpu::recorder::init_recorder();
-        let mut resource_metering_reporter_worker =
-            Box::new(LazyWorker::new("resource-metering-reporter"));
+        let mut resource_metering_reporter_worker = Box::new(
+            WorkerBuilder::new("resource-metering-reporter")
+                .pending_capacity(30)
```

Comment:
Does this mean that we will have maximum 30 seconds' data to report when the report remote is blocked? And what will happen if the worker is full?

Reply:
Basically, nothing will block the reporter worker except the runtime is busy, so it's hard to be full. If full, new coming per-second records are dropped.

The RPC call is spawned to grpc runtime at every tick (/minute). It's limited to be at most one RPC call is in-flight at this PR. If the report remote is blocked, it won't call RPC at later ticks.